h5bench_patterns/h5bench_read.c

```diff
@@ -200,6 +200,143 @@ _set_dataspace_seq_2D(hid_t *filespace_in_out, hid_t *memspace_out, unsigned lon
     return dim_1 * dim_2;
 }
 
```

Comment:
Minor: Add a short description here and at the start of the remaining functions describing what this function does.

Reply:
Fixed